src/DataStructures.jl

```diff
@@ -38,6 +38,7 @@ module DataStructures
     export MutableBinaryHeap, MutableBinaryMinHeap, MutableBinaryMaxHeap
     export heapify!, heapify, heappop!, heappush!, isheap
     export BinaryMinMaxHeap, popmin!, popmax!, popall!
+    export MutableBinomialHeap, MutableBinomialHeapNode, push!, show, top, top_with_handle, minimum, _union!, update!, delete!, HeapBoundsError, find_node
```

Comment:
We do not need to reexport things we overload from `Base`, 

Also what is up with experingign `_union!` ?

Reply:
_union! is used to merge two binomial heaps